README.md

```diff
@@ -1,13 +1,13 @@
 # prebuild-install [![Build Status](https://travis-ci.org/mafintosh/prebuild-install.svg?branch=master)](https://travis-ci.org/mafintosh/prebuild-install)
 
-A command line tool for easily install prebuilds for multiple version of node/iojs on a specific platform.
+Command line tool to easily install prebuilds for multiple version of node/iojs
```

Comment:
I reworded the sentence so it can fit the limit of 80 characters, bigger titles don't show well both on markdown code and when rendered on Github